source/js/components/nav/desktop-dropdown.js

```diff
@@ -115,7 +117,23 @@ class NavDesktopDropdown extends Accordion {
 
     this.closeAccordion(this.accordion, this.titleButton, this.content);
     this.closeSiblings();
+
+    document.removeEventListener("scroll", this.handleScroll);
+  }
+
+  scrollListener() {
+    const newTopOffset = document
```

Comment:
let's be extra careful to prevent runtime error

```js
const newFocusInNav =
  event.relatedTarget &&
  document.querySelector(".wide-screen-menu")?.contains(event.relatedTarget);
```

Reply:
Since we're listening to the document's scroll event, I only needed to confirm the element is in the DOM.